editor/plugins/asset_library_editor_plugin.h

```diff
@@ -305,6 +307,8 @@ class EditorAssetLibrary : public PanelContainer {
 
 	void _install_external_asset(String p_zip_path, String p_title);
 
+	int _asset_items_column_width;
```

Comment:
```suggestion
	int asset_items_column_width;
```
While having special prefixes for member variables would be desirable we don't prefix with `_` normally

Reply:
I amended the commit with this change introduced. This is good to review now.